query_execution/WorkOrdersContainer.hpp

```diff
@@ -46,10 +48,14 @@ class WorkOrdersContainer {
    *
    * @param num_operators Number of operators in the query DAG.
    * @param num_numa_nodes Number of NUMA nodes in the system.
+   * @param query_id The ID of the query.
    **/
   WorkOrdersContainer(const std::size_t num_operators,
-                      const std::size_t num_numa_nodes)
-      : num_operators_(num_operators), num_numa_nodes_(num_numa_nodes) {
+                      const std::size_t num_numa_nodes,
+                      const std::size_t query_id)
```

Comment:
I think we have a container per query, so `query_id` is the most significant argument, then some info for this query, i.e., `num_operators`.